src/genie/libs/parser/asa/show_route.py

```diff
@@ -326,3 +326,156 @@ def cli(self, output=None):
                 index += 1
 
         return ret_dict
+
+
+# =============================================
+# Parser for 'show route bgp'
+# =============================================
+class ShowRouteBgp(ShowRouteSchema):
```

Comment:
It's something I probably should have asked myself before I reviewed the rest of this parser, but couldn't you have just added this to the `ShowRoute` class instead of creating a new one?